z/zookeeper/Dockerfiles/3.6.3_ubi8/README.md

```diff
@@ -0,0 +1,8 @@
+# Zookeeper Build steps
+```
+
+docker build -t zookeeper .
```

Comment:
Either tag  it as `zookeeper-ubi8` 
or
run as `zookeeper:latest`

Reply:
Updated to zookeeper:latest